docs/tutorials/local_setup.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: "Quickstart"
-nav-title: '<i class="fa fa-power-off title appetizer" aria-hidden="true"></i> Quickstart'
-nav-parent_id: root
-nav-pos: 2
+title: "Local Setup"
+nav-title: 'Local Setup'
+nav-parent_id: setuptutorials
+nav-pos: 10
 ---
 <!--
 Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
True, that page hasn't been updated for a while. I don't think it's critical (the WebUI hasn't changed that much since then), but it would be nice to update.
I'd consider this as a follow up issue.